src/libs/ReportUtils.js

```diff
@@ -3625,6 +3625,46 @@ function getReportPreviewDisplayTransactions(reportPreviewAction) {
     );
 }
 
+/**
+ * Return iou report action display message
+ *
+ * @param {Object} reportAction report action
+ * @returns {String}
+ */
+function getIOUReportActionDisplayMessage(reportAction) {
+    const originalMessage = _.get(reportAction, 'originalMessage', {});
+    let displayMessage;
+    if (originalMessage.type === CONST.IOU.REPORT_ACTION_TYPE.PAY) {
+        const {amount, currency, IOUReportID} = originalMessage;
```

Comment:
Coming from https://github.com/Expensify/App/issues/29441

The `REPORT_ACTION_TYPE.PAY` action type is used for both fulfilling existing requests and sending money. To differentiate between these two scenarios, we should have checked if the `originalMessage` contains the `IOUDetails` property. If it does, it indicates that this is a 'Send money' action.